src/config/dynamic-theme-fixes.config

```diff
@@ -111,6 +111,9 @@ CSS
 .catend {
     background: ${#e8e8e8} !important;
 }
+div.description > span {
+    height: 0 !important;
```

Comment:
`display: none` Should be understandable why it isn't displaying.

Reply:
That works too.